k9mail/src/main/res/values/strings.xml

```diff
@@ -553,6 +553,9 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="account_settings_strip_signature_label">Strip signatures on reply</string>
     <string name="account_settings_strip_signature_summary">Signatures will be removed from quoted messages</string>
 
+    <string name="account_settings_bundle_message_followings_label">Bundle message followings</string>
```

Comment:
I don't like the setting's name. Thunderbird's "Place replies in the folder of the message being replied to" is better but also very long. Suggestions are welcome.

Reply:
"Save replies in same folder as original"?